src/eth/TransactionObject.js

```diff
@@ -9,12 +9,14 @@ export default class TransactionObject extends TransactionLifeCycle {
   constructor(
     transaction,
     web3Service,
+    nonceService,
```

Comment:
there's another use of `new TransactionObject` in `OasisExchangeService` which may need to be modified to have this passed into it